README.md

```diff
@@ -2,47 +2,64 @@
 
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/db5eda4c50b24c008b2c19f1117c9cef)](https://www.codacy.com/gh/codacy/codacy-coverage-reporter-action/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=codacy/codacy-coverage-reporter-action&amp;utm_campaign=Badge_Grade)
 
-> 🤖 Automated multi-language coverage reporter for [Codacy](https://www.codacy.com/).  
-> For more information about Codacy see our [5 minutes quickstart](https://docs.codacy.com/getting-started/codacy-quickstart/).
+GitHub Action for [uploading coverage reports to Codacy](https://docs.codacy.com/coverage-reporter/#uploading-coverage) on all commits and pull requests, tracking code coverage, and returning code coverage issues.
```

Comment:
As we say, the Codacy Coverage Reporter (and subsequently, the GitHub Action) acts just as an uploader of the coverage reports to Codacy. This part should still mention the added benefits of uploading coverage to Codacy, but it must be rewritten. It could be something like this:

```suggestion
GitHub Action for [uploading coverage reports to Codacy](https://docs.codacy.com/coverage-reporter/#uploading-coverage) on all commits and pull requests, allowing you to track code coverage on your repository and to flag pull requests that cause coverage to drop or add code that isn't tested.
```